project/make/.dockerinit_gccgo

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Can we please rename this file to `.dockerinit-gccgo` instead so it matches the rest of the files in this directory?